### ISSUE TITLE
How do I set reminders for events via Nylas API?

### ISSUE THREAD

**Issue by abdullahtariq1171** [nylas/nylas-nodejs#390]
**Is your feature request related to a problem? Please describe.**
How do I set reminders for events via Nylas API? 

If this is not supported, how can I set it directly on google calendar bypassing Nylas? A quick code snippet would be very helpful.

**Describe the solution you'd like**
Support reminders attributes to match with google https://developers.google.com/calendar/api/v3/reference/events

```
"reminders": {
    "useDefault": boolean,
    "overrides": [
      {
        "method": string,
        "minutes": integer
      }
    ]
  }

```

**Describe alternatives you've considered**
I haven't.

**Additional context**
I believe other packages support it https://github.com/nylas/nylas-ruby/pull/370 (edit: fixed link)
I also saw this one related attribute has been added recently https://github.com/nylas/nylas-nodejs/pull/361

If this is work in progress, Could you share pragmatic ETA? We're using Nylas on our production app, any information would be helpful.

**Comment by abdullahtariq1171** [comment]
Basically I'm looking closed to following use-case

```
reminders: {
    useDefault: false,
    overrides: [
      { method: "email", minutes: 24 * 60 },
      { method: "popup", minutes: 10 },
    ],
  },
```

**Comment by mrashed-dev** [comment]
Hey @abdullahtariq1171! Sorry for the long delay on this answer but it should be available since v6.6.0. The implementation was completed here #399. Here's how to use the implementation:

```java
event.reminders = new EventReminder({
  reminderMinutes: '[20]',
  reminderMethod: 'popup',
});
```

Again apologies for the delay but I hope this resolves your issue. If not feel free to reopen it and ping me. Thanks!
